index.html

```diff
@@ -80,7 +80,7 @@ <h1>Bath Institute for Rheumatic Disease</h1>
                 </div>
                 <div class="work-card">
                     <div class="meta">
-                        <img src="assets/img/5.jpg" class="photo"></img>
+                        <img src="assets/img/33.jpg" class="photo"></img>
```

Comment:
i couldn't see your portfolio images?

Reply:
i'll make sure this is working in the work branch :)